lib/git_reflow/git_server/pull_request.rb

```diff
@@ -35,13 +53,48 @@ def reviewers_pending_response
         reviewers - approvals
       end
 
+      def check_approvals
+        has_comments_or_approvals = (has_comments? or approvals.any?)
+
+        case self.class.minimum_approvals
+        when "0"
+          true
+        when "", nil
+          # Approvals from every commenter
+          has_comments_or_approvals && reviewers_pending_response.empty?
+        else
+          approvals.size >= self.class.minimum_approvals.to_i and !last_comment.match(self.class.approval_regex).nil?
```

Comment:
@codenamev I am confused by this.  `self.class.minimum_approvals.to_i > 0` would be `true` for `minimum_approvals` config value of `""`, which is our indicator of *all commenters must approve*.  Is this due to some shortcoming of the bitbucket API?  Can we not get the body of the comment from bitbucket?